### PR TITLE
Notebook reorganization

### PR DESCRIPTION
**Pull request by kfogelso**
Addressing issues #24 #25 

I also documented error messages (mostly minor) I came across while running through the various notebooks, which we can go over and assign at a later date. 

@ahdilmore @cguccione @antgonza

### CODE REVIEW

**Comment by antgonza** [comment]
@kfogelso, if I understand this correctly, the goal of this PR is to move files around and add testing data. The first is easy to see but I'm a bit confused about the second: it seems like there is not huge difference between the [action running time](https://github.com/biocore/metagenomics_pooling_notebook/actions) between your PR and previous ones, which to me it means there are no new tests being run with the test data. However, it seems like there is a slight [increment in coverage](https://github.com/biocore/metagenomics_pooling_notebook/runs/4542592503?check_suite_focus=true#step:7:26); could you clarify this?

Thank you.

**Comment by kfogelso** [comment]
@antgonza from what I understand, for #25 @ElDeveloper wanted me to run cell by cell through every notebook and document if/where I'm getting errors, as well as if/where there are test data/test outputs missing. Since last running through the notebooks, it looks like some of this issues I was seeing previously have been addressed. However, I identified one needed test_data file not on the repo (in metagenomics_repooling notebook ./finrisk_data/complete_dataframes/FinRisk_5-8_back-compiled_dummy.txt). Was planning on talking through minor error messages w/ @cguccione and @ahdilmore at a later point to see if it these are thing that can be quickly addressed. 

Does this answer your question?

**Comment by antgonza** [comment]
It does, thank you for the explanation.

**Comment by antgonza** [comment]
Forgot to add, then I think it seems fine but I think it will be better if we wait for @ElDeveloper feedback.

**Comment by antgonza** [comment]
@ElDeveloper, friendly ping about this.

File: notebooks/metagenomics_samplesheet_generator.ipynb [hunk 1]
```diff
@@ -829,9 +904,17 @@
...
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[31mErrorMessage: \u001b[0mIn the Bioinformatics section Project #1 does not have experiment_design_description specified\n"
```
Comment (ElDeveloper) [inline notebooks/metagenomics_samplesheet_generator.ipynb:914]: Can you add a sample experiment design description in this cell?

File: notebooks/metagenomics_samplesheet_generator.ipynb [hunk 2]
```diff
@@ -925,43 +1009,18 @@
...
-      "[Data]\r\n",
-      "Lane,Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,I5_Index_ID,index2,Sample_Project,Description\r\n",
-      "1,33-A1,33-A1,Example Plate 1,A1,iTru7_109_01,CTCGTCTT,iTru5_19_A,TGTGCGTT,Example Project,33-A1\r\n",
-      "1,820072905-2,820072905-2,Example Plate 1,C1,iTru7_109_02,CGAACTGT,iTru5_19_B,CTAGGCAT,Example Project,820072905-2\r\n",
-      "1,820029517-3,820029517-3,Example Plate 1,E1,iTru7_109_03,CATTCGGT,iTru5_19_C,TCCGTATG,Example Project,820029517-3\r\n",
-      "1,820073753-4,820073753-4,Example Plate 1,G1,iTru7_109_04,TCGGTTAC,iTru5_19_D,TAGTGACC,Example Project,820073753-4\r\n",
-      "1,820049719-1,820049719-1,Example Plate 1,I1,iTru7_109_05,AAGTCGAG,iTru5_19_E,CGGAATAC,Example Project,820049719-1\r\n",
-      "1,33-F1,33-F1,Example Plate 1,K1,iTru7_109_06,TATCGGTC,iTru5_19_F,AAGAGCCA,Example Project,33-F1\r\n",
-      "1,820048473-9,820048473-9,Example Plate 1,M1,iTru7_109_07,TATTCGCC,iTru5_19_G,CGATAGAG,Example Project,820048473-9\r\n"
+     "ename": "AttributeError",
```
Comment (ElDeveloper) [inline notebooks/metagenomics_samplesheet_generator.ipynb:1016]: Can you rerun this cell after fixing the previous error?

File: notebooks/metagenomics_samplesheet_generator.ipynb [hunk 3]
```diff
@@ -985,9 +1044,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "TypeError",
```
Comment (ElDeveloper) [inline notebooks/metagenomics_samplesheet_generator.ipynb:1051]: Can you rerun this cell after fixing the previous error?

**Review by ElDeveloper — COMMENTED** [review]
Thanks just a couple requests: `notebooks/iTru index combo generator.ipynb` should be deleted and there's some errors in the metagenomics sample sheet generator but otherwise this looks good to be merged as is. @charles-cowart any chance you can lend @kfogelso a hand to fix these?

File: notebooks/metagenomics_samplesheet_generator.ipynb (hunk 1 above)
Comment (charles-cowart) [reply notebooks/metagenomics_samplesheet_generator.ipynb:914]: @ElDeveloper sure I can take a look in the morning

**Comment by antgonza** [comment]
Thank you everyone. Just for the record: @kfogelso and I met and addressed @ElDeveloper comments so I think is fine to merge.

**Pull request merged**